File: web/js/gaiaModel/Recursos.js
/* 
 * Classe RECURSOS
 */

function Recursos(nome, tipo, arquivo)
{
        if(nome.Nome == undefined)
        {
                this.Nome = nome;
                this.Tipo = tipo;
                this.Arquivo = arquivo;
                this.cod = 2147483647;
                this.superCod = 2147483647;
        }
        else
        {
                this.Nome = nome.Nome;
                this.Tipo = nome.Tipo;
                this.Arquivo = nome.Arquivo;
                this.cod = nome.cod;
                this.superCod = nome.superCod;
        }
}

File: src/java/Gaia/model/Recursos.java
package Gaia.model;

import mpro.MproEntity.MproEntityRelation;
import org.bson.types.ObjectId;

/**
 *
 * @author matheus
 */
public class Recursos extends MproEntityRelation
{
        public String Nome;
        public int Tipo;
        public String Arquivo;
        
        public Recursos() {}
        
        public Recursos(String nome, int tipo, String arquivo)
        {
                this.Nome = nome;
                this.Tipo = tipo;
                this.Arquivo = arquivo;
        }
        
        public void resetAllIds()
        {
            /* Recursos are used in recursoInt and have to be a same cod
               than the copied project */
            //this.cod = new ObjectId();
        }
}
